inst/tests/tests.Rraw

```diff
@@ -18419,3 +18419,8 @@ test(2251.10, dim(fread(text, fill=TRUE)), c(9L, 9L))
 test(2251.11, dim(fread(text, fill=7)), c(9L, 9L))
 test(2251.12, dim(fread(text, fill=9)), c(9L, 9L))
 test(2251.13, dim(fread(text, fill=20)), c(9L, 20L)) # clean up currently only kicks in if sep!=' '
+
+.datatable.aware = FALSE
+dt = data.table(a = 1L)
+test(2252.1, dt[, b:=2L], error = "^\\[ was called on a data.table in an environment that is not data.table-aware.*")
+test(2252.2, dt[, b:=2L], error = "^\\[ was called on a data.table in an environment that is not data.table-aware.*")
```

Comment:
The same test is duplicated AFAIU - maybe the second case should be `let`. Also, we should unset the .datatable.aware variable. Otherwise, once we add more tests which expect to be a data.table aware package, it will fail and cause some confusion. Another alternative is to evaluate it in a different environment context but I am not sure if that is necessary.

The easiest way to do it is just remove the variable (e.g., `rm(.datatable.aware)`).